src/utils/update/review_trusts.ts

```diff
@@ -105,3 +105,18 @@ export const setTrusteesAsReviewed = (appData: ApplicationData, trusteeType: Tru
         return false;
   }
 };
+
+export const moveTrustOutOfReview = (appData: ApplicationData) => {
+  const trustIndex = (appData.update?.review_trusts ?? []).findIndex(trust => trust?.review_status?.in_review);
+  const trust = appData.update?.review_trusts?.splice(trustIndex, 1)[0];
```

Comment:
findIndex/splice is a little bit hard to understand. Maybe a straight forward loop or foreach()?

Reply:
I see splice used elsewhere so ok.